newhelm/caching.py

```diff
@@ -70,7 +77,7 @@ def get_cached_response(self, request):
     def update_cache(self, request, response):
         if not self._can_encode(request) or not self._can_encode(response):
             return
-        encoded_request = self._encode_request(request)
+        encoded_request = self._hash_request(request)
```

Comment:
That's a bit of an odd variable name; maybe `cache_key` or just inlining it.

Reply:
Renamed `encoded_request` to `cache_key`.